client/blocks/taxonomy-manager/style.scss

```diff
@@ -66,6 +66,13 @@
 	}
 }
 
+.taxonomy-manager__count {
```

Comment:
I prefer to use `.taxonomy-manage .count {}` in these cases, as it shows more clearly the component -> child relationship and ensures your styles take precedence over default `.count` styles.